example/examples/OpenWindow.tsx

```diff
@@ -0,0 +1,117 @@
+import React, {Component} from 'react';
+import {Button, Switch, StyleSheet, Text, View} from 'react-native';
+
+import WebView from 'react-native-webview';
+
+const HTML = `
+<!DOCTYPE html>\n
+<html>
+  <head>
+    <title>Alerts</title>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=320, user-scalable=no">
+    <style type="text/css">
+      body {
+        margin: 0;
+        padding: 0;
+        font: 62.5% arial, sans-serif;
+        background: #ccc;
+      }
+      a {
+        display: block;
+        margin-top: 20px;
+      }
+    </style>
+  </head>
+  <body>
+    <button onclick="openWindow()">Call window.open() from JS</button>
+    <a href="https://example.com" target="_blank">Call target=_blank link from DOM</a>
+    <script>
+      function openWindow() {
+        window.open('https://example.com', '_blank')
+      }
+    </script>
+  </body>
+</html>
+`;
+
+type Props = {};
+type State = {
+  shouldInterceptOpenWindow: boolean,
+  text: string,
+  webViewKey: number
+};
+
+export default class Alerts extends Component<Props, State> {
```

Comment:
This class name should be updated 🙂

Reply:
Fixed in https://github.com/react-native-webview/react-native-webview/compare/5e7a71e0ea21bc0d1eeaa492683a306d9fbea576..1b03904b3c44b0e31fa7e7108bae9b30c1af1418